test/functional/lua/vim_spec.lua

```diff
@@ -1297,7 +1297,7 @@ describe('lua stdlib', function()
     matches('aaabbbcccddd', res)
   end)
 
-  it('vim.call, vim.fn', function()
+  describe('vim.call, vim.fn', function()
     eq(true, exec_lua([[return vim.call('sin', 0.0) == 0.0 ]]))
```

Comment:
i suggest moving this code into an `it()` (instead of putting `eq()` directly inside a `describe()`), because:

- it's unusual/inconsistent
- am not confident that busted will make any sense of it if such a thing were to fail.
- also not sure how our "test id" counter would handle it.

Reply:
Seems like this was a good idea. the tests would fail but was prefixed with "nan" instead of `Txx`. I moved the new test to its own "it" as I noticed other vim.fn tests existed outside this block anyway.